tests/Endpoints/WebhookTest.php

```diff
@@ -41,6 +49,9 @@ public function testCreate(): void
          */
         self::assertSame('http://sdktest.local', $webhook->getUrl());
         self::assertCount(1, $webhook->getHeaders() ?? []);
```

Comment:
This is a weak test on headers, it asserts that the headers array has one value in it, but it could be anything at all.

Reply:
Didn't notice that as the test was already there. Agreed, will change the test.